### PR TITLE
Add  command for booting to live image

### PR DESCRIPTION
**Pull request by ryg-**
Added command to boot to live maintenance os image for allowing it to use as a parking state for teuthology.
lib/pxelinux_cfg.py  - fixed menu generation, added configuration reading
README.md - improved documentation
bin/pelagos.py - added code for pointing on maintenance image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,16 +174,25 @@ For manual node configuration could be used salt commands:
...
+
+* 'os' - os version which used for boot. It compatible with
+        teuthology format, e.g. could be ' sle-15.1'. Also
+        supported 2 special 'os'-es:
+         - 'local' - boot for local disk
+         - 'maintenance_image' - maintenance disk ram-only image, also it
+            used as 'parking' image in unlock command
+
+
+* 'node' -
```
Comment (kshtsk) [inline README.md]: I guess you missed a description here.

File: README.md [hunk 2]
```diff
@@ -174,16 +174,25 @@ For manual node configuration could be used salt commands:
...
 
 For provision one node for testing boot could be use cmd.
 
-        curl -i http://<server ip>:5000/node/provision -X POST -d {"os":"<dir name from tftp>", "node":"<nodename from configuration>"}
+    curl -i http://<server ip>:5000/node/provision -X POST \
+    -F 'os=_distro name_' -F 'node=_node name from cfg_'
```
Comment (kshtsk) [inline README.md]: You better use referential name to the description above.

File: README.md [hunk 3]
```diff
@@ -174,16 +174,25 @@ For manual node configuration could be used salt commands:
 ## Provision
 
 Warning! curl commands is subject for change
-    os = request.form['os']
-    node_id = request.form['node']
+
+* 'os' - os version which used for boot. It compatible with
+        teuthology format, e.g. could be ' sle-15.1'. Also
```
Comment (kshtsk) [inline README.md]: could be => can be?

File: lib/pxelinux_cfg.py [hunk 4]
```diff
@@ -93,14 +108,21 @@ def get_macfile(node):
...
+                             maintenance_image_initrd,
+                             default_pxe_server)
     else:
         image = re.compile('^oem-').sub('', os_id)
         image = re.compile('-\d+\.\d+\.\d+').sub('', image)
-        cfg = pxe_common_os_boot_tmpl.format(os_id, os_id,
-                                             default_pxe_server, os_id, image)
+        cfg = pxe_common_os_boot_tmpl.format(
+                            os_id, os_id,
+                            default_pxe_server, os_id, image)
```
Comment (kshtsk) [inline lib/pxelinux_cfg.py]: This is kind of magic using same argument as os_id, can this format has a named arguments for replacement?

File: README.md (hunk 1 above)
Comment (ryg-) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (ryg-) [reply README.md]: done

File: lib/pxelinux_cfg.py (hunk 4 above)
Comment (ryg-) [reply lib/pxelinux_cfg.py]: done

File: README.md [hunk 5]
```diff
@@ -174,16 +174,25 @@ For manual node configuration could be used salt commands:
...
+            used as 'parking' image in unlock command
+
+
+* 'node' - node name as defined in pelagos configuration
 
 For provision one node for testing boot could be use cmd.
 
-        curl -i http://<server ip>:5000/node/provision -X POST -d {"os":"<dir name from tftp>", "node":"<nodename from configuration>"}
+    curl -i http://<server ip>:5000/node/provision -X POST \
+    -F 'os=os' -F 'node=node'
```
Comment (kshtsk) [inline README.md]: nit:
```
 -F 'os=<os>' -F 'node=<node>'
```

Comment (ryg-) [reply README.md]: done

**Pull request merged**